controllers/articles.controllers.js

```diff
@@ -0,0 +1,16 @@
+const { fetchArticleById } = require("../models/articles.models");
+
+exports.getArticleById = async (req, res, next) =>{
+    const id = req.params.article_id
+
+    try {
+        const article = await fetchArticleById(id);
+        res.status(200).send({article})
+    } catch(err){
+        if (err.code === '22P02'){
```

Comment:
This really should be in an error handler, not on a specific endpoint controller (you'll have to duplicate it a lot then you see)